File: talleres java/operaciones.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        // Solicitar los valores de A y B
        System.out.print("Ingrese valor para A: ");
        int A = scanner.nextInt();
        
        System.out.print("Ingrese valor para B: ");
        int B = scanner.nextInt();
        
        // Realizar las operaciones y mostrar los resultados
        
        // Suma
        int suma = A + B;
        System.out.println("La suma de los números es: " + suma);
        
        // Resta
        int res = A - B;
        System.out.println("La resta de los números es: " + res);
        
        // Multiplicación
        int multi = A * B;
        System.out.println("La multiplicación de los números es: " + multi);
        
        // División
 

File: GO/operaciones.go
package main

import "fmt"

func main() {
	// Definir las variables para los valores de entrada
	var A, B int

	// Solicitar los valores al usuario
	fmt.Print("Ingrese valor: ")
	fmt.Scanln(&A)

	fmt.Print("Ingrese valor: ")
	fmt.Scanln(&B)

	// Realizar las operaciones
	suma := A + B
	fmt.Println("La suma de los números es:", suma)

	res := A - B
	fmt.Println("La resta de los números es:", res)

	multi := A * B
	fmt.Println("La multiplicación de los números es:", multi)

	// Aseguramos que la división sea de tipo float64 para manejar decimales
	div := float64(A) / float64(B)
	fmt.Println("La división de los números es:", div)

	// Comparaciones
	igual := A == B
	fmt.Println("Los números son iguales:", igual)

	// Mayor y menor
	mayor := A > B
	fmt.Println("El número mayor es:", A)
	fmt.Println("El número menor es:", B)

	// Mostrar el resultado de mayor
	if mayor {
		fmt.Println("El número mayor es:", A)
	} else {
		fmt.Println("El número mayor es:", B)
	}
}
